src/Microsoft.Azure.EventHubs.Processor/ICheckpointManager.cs

```diff
@@ -49,8 +49,9 @@ public interface ICheckpointManager
         /// <summary>
         /// Update the checkpoint in the store with the offset/sequenceNumber in the provided checkpoint.
         /// </summary>
+        /// <param name="lease">Partition information against which to perform a checkpoint.</param>
         /// <param name="checkpoint">offset/sequeceNumber to update the store with.</param>
-        Task UpdateCheckpointAsync(Checkpoint checkpoint);
+        Task UpdateCheckpointAsync(Lease lease, Checkpoint checkpoint);
```

Comment:
Can we deprecate this signature, and add the new one? That will allow us to release this a minor change according to SemVer.